### PR TITLE
add cred report to iam users

### PR DESCRIPTION
**Pull request by roneli**

### CODE REVIEW

**Review by fdistorted — CHANGES_REQUESTED** [review]
Everything looks great! I found a few minor suggestions. Please take a look.

File: resources/iam_users.go [hunk 1]
```diff
@@ -350,11 +335,74 @@ func fetchIamUserAttachedPolicies(ctx context.Context, meta schema.ClientMeta, p
...
 	}
 	return resource.Set("tags", tags)
 }
+
+type wrappedUser struct {
+	types.User
+	*reportUser
+}
+
+type reportUser struct {
```
[the remark this answers is not in the document]
Comment (roneli) [reply resources/iam_users.go:443]: added

**Comment by roneli** [comment]
fixes some https://github.com/cloudquery/cloudquery/issues/95

File: resources/iam_users.go [hunk 2]
```diff
@@ -246,38 +214,55 @@ func fetchIamUsers(ctx context.Context, meta schema.ClientMeta, parent *schema.R
...
+	r := resource.Item.(wrappedUser)
+	if r.reportUser == nil {
+		return nil
+	}
+	location, err := time.LoadLocation("UTC")
+	if err != nil {
+		return err
+	}
+
+	if enabled, err := cast.ToBoolE(r.PasswordStatus); err != nil {
```
[the remark this answers is not in the document]
Comment (roneli) [reply resources/iam_users.go]: fixed

**Pull request merged**